client/src/controllers/ActionController.test.js

```diff
@@ -86,6 +86,58 @@ describe('ActionController', () => {
     });
   });
 
+  describe('click method for textarea', () => {
```

Comment:
I have removed this test block, it's testing something that's already in the test suite.